test/HasPredicateScanTest.cpp

```diff
@@ -331,11 +331,11 @@ TEST(CountAvailablePredicates, patternTrickTest) {
   CompactVectorOfStrings<Id> hasRelation(hasRelationSrc);
   CompactVectorOfStrings<Id> patterns(patternsSrc);
 
-  RuntimeInformation runtimeInfo;
+  auto runtimeInfo = std::make_shared<RuntimeInformation>();
   try {
-    CALL_FIXED_SIZE(
-        input.numColumns(), CountAvailablePredicates::computePatternTrick,
-        input, &result, hasPattern, hasRelation, patterns, 0, &runtimeInfo);
+    CALL_FIXED_SIZE(input.numColumns(),
+                    CountAvailablePredicates::computePatternTrick, input,
+                    &result, hasPattern, hasRelation, patterns, 0, runtimeInfo);
```

Comment:
Similar here, we don't contribute to the ownership, so no need to pass shared_ptrs.